core/src/components/footer/footer.tsx

```diff
@@ -94,6 +123,9 @@ export class Footer implements ComponentInterface {
   render() {
     const { translucent, collapse } = this;
     const mode = getIonMode(this);
+    const tabs = this.el.closest('ion-tabs');
+    const tabBar = tabs ? tabs.querySelector('ion-tab-bar') : null;
```

Comment:
Can the tab bar selector be simplified to:
```ts
const tabBar = tabs?.querySelector('ion-tab-bar');
```

or do you need to the type signature to be `HTMLElement | null` instead of `HTMLElement | null | undefined`?